src/api/siteService.jsx

```diff
@@ -0,0 +1,42 @@
+const API_BASE_URL = '/api/v1';
+
+// GET /api/v1/site
+export async function getSites() {
+    const response = await fetch(`${API_BASE_URL}/site`);
```

Comment:
Switch the javascript fetch function with the axios one. You can see an example in api/AuthService.jsx